testing/libbpf/APKBUILD

```diff
@@ -1,7 +1,6 @@
 # Maintainer: Adam Jensen <acjensen@gmail.com>
 pkgname=libbpf
-pkgver=5.0.8
-_versionref="816253000e918f22e3b395d1bf743503d3508b41"
+pkgver=0.0.3
```

Comment:
Won't that fail as 5.0.8 is bigger than 0.0.3 ?

Reply:
My bcc package is the only one that depends on libbpf, so I think we're safe. What would fail?

Reply:
People using libbpf-dev won't get an update, builders will use libbpf-5.0.8 instead of libbpf-0.0.3 to compile stuff up until the new version scheme reaches a version bigger than 5.0.8.

Trying to install libbpf after compiling your version gives me 

```
(1/2) Installing elfutils (0.168-r2)
(2/2) Installing libbpf (5.0.8-r0)
```

So it would fail as it would make a package with a version number lower than the current one making it never a viable candidate to be used.

Reply:
I understand, but I think it's acceptable to make the breaking change. This package is new, and probably no one is using it (or libbpf-dev) yet since it's in the testing repo. Is it possible to purge the existing 5.0.8-r0 version from the index?